draft-ietf-rats-corim.md

```diff
@@ -380,6 +380,13 @@ that MUST have a different identifier.
 
 For an example profile definition, see {{-psa-endorsements}}.
 
+A tag (e.g., CoMID, CoBOM) MAY specify a profile identifier to take precedence for the interpretation of the remainder of the tag contents if the following requirements hold:
+
+*  Any interpretation of the tag that depends on contents from the `corim-map` containing the tag have the same meaning across the tag's profile and the `corim-map.profile`.
+*  The `corim-map.profile` permits (exclusively or by omission) the tag to to use the profile.
+
```

Comment:
Also worth specifying explicitly that the scope of the profile defined at the tag level is ONLY restricted to the tag where it is defined. If no profile is defined in the tag then the CoRIM profile should be used for understanding the semantics

Reply:
"for the interpretation of the remainder of the tag contents if the following requirements hold:"

I think this is handled?